src/main/java/com/iota/iri/controllers/ApproveeViewModel.java

```diff
@@ -179,16 +179,20 @@ private static void cacheDelete(Tangle tangle, Indexable hash) {
      */
     private static void cacheRelease(Tangle tangle) throws Exception {
         Cache<Indexable, ApproveeViewModel> cache = tangle.getCache(ApproveeViewModel.class);
-        Queue<Indexable> releaseQueueCopy = cache.getReleaseQueueCopy();
+        Queue<Indexable> releaseQueue = cache.getReleaseQueue();
 
-        for (int i = 0; i < cache.getConfiguration().getReleaseCount(); i++) {
-            Indexable hash = releaseQueueCopy.poll();
+        int releaseCount = 0;
```

Comment:
I think we can even do without this `releaseCount` because we know what it will be thanks to `ReleaseCount`

Reply:
`releaseCount` is the number of items released. We need it to make sure we only release the specified `config.getReleaseCount` items.

Reply:
If you are suggesting instead that we should remove the whole release count feature and have the cache at `1k/1`, then once the cache is full, it'll do 1 `release` before every `put`.
Is that not more expensive?

Reply:
It means that for every put we do a release, yes
I am not sure why it would be more expensive... It would smooth out cpu effort over time
If you're sure that it is more expensive, please explain here

If not you can try it out :-)